.gitignore

```diff
@@ -23,6 +23,7 @@ debug
 #Various temporary created while building/running jx.
 /build/
 release/
+!pkg/cmd/step/update/release/
```

Comment:
err, what?

Reply:
yeah, that was a quick fix... the step folder is 'release/' what was being ignored by git due to line above

Reply:
I'll look to update that to something more sensible

Reply:
I meant to go back and change it, but forgot... thanks @pmuir